smartsim/launchable/job.py

```diff
@@ -158,3 +159,35 @@ def __str__(self) -> str:  # pragma: no cover
         string = f"SmartSim Entity: {self.entity}\n"
         string += f"Launch Settings: {self.launch_settings}"
         return string
+
+
+@t.final
+class Record:
+    """A record a job that was launched and launch ID assigned to the
```

Comment:
`A record a job that was launched and launch ID assigned to the launching` -> `A Record object to track a launched job along with its assigned launch ID`?

Reply:
Every time I read my own writing, I become more and more convinced that I'm not actually fluent in English, hahaha. Updated to make it coherent!